### PR TITLE
Creates Unit test to CmpJpaConversion

### PR DESCRIPTION
**Pull request by otaviojava**

### CODE REVIEW

File: container/openejb-core/src/test/java/org/apache/openejb/config/CmpJpaConversionTest.java [hunk 1]
```diff
@@ -0,0 +1,179 @@
+/*
```
Comment (jgallimore) [inline container/openejb-core/src/test/java/org/apache/openejb/config/CmpJpaConversionTest.java]: Thanks for the extra tests, I'll have a go with them shortly.

**Comment by otaviojava** [comment]
@jgallimore when you have time, could you review/merge this PR?

**Comment by dblevins** [comment]
Please create a JIRA, ask for it to be assigned on the dev list and update the community via the dev list on what problem this PR solves and what people should know so they can review it.

**Comment by otaviojava** [comment]
@dblevins done
https://issues.apache.org/jira/browse/TOMEE-2492?jql=text%20~%20%22CmpJpaConversion%22

**Comment by dblevins** [comment]
Can you also do the second part, which is update the community via the dev list on what problem this PR solves and what people should know so they can review it.

**Pull request closed without merging**